src/screens/login/Login.jsx

```diff
@@ -95,16 +104,13 @@ const Login = ({ navigation }) => {
             authenticate(email, password);
           }}
         >
-          <Typography style={styles.textFontLogin} id="LOGIN" />
+          <Typography style={globalStyles.buttonText} id="LOGIN" />
         </TouchableOpacity>
       </View>
 
       <View style={styles.guestView}>
         <TouchableOpacity style={styles.guest}>
-          <Typography
-            id="CONTINUE_AS_GUEST"
-            style={[styles.fontBasicText, { color: '#2C2C2C80' }]}
-          />
+          <Typography id="CONTINUE_AS_GUEST" style={[globalStyles.basicText]} />
```

Comment:
```suggestion
          <Typography id="CONTINUE_AS_GUEST" style={globalStyles.basicText} />
```
Seems to be the last style to fix